src/components/my-career/SkillsRecommendationCourses.jsx

```diff
@@ -74,14 +74,12 @@ const SkillsRecommendationCourses = ({ index, subCategoryName, subCategorySkills
     },
     [filters, index, skillsFacetFilter],
   );
-
   if (hitCount === 0) {
     return null;
   }
-
   return (
     <div>
-      <h5 className="mb-3 mt-n4">More courses that teach you {subCategoryName} Skills</h5>
+      {subCategoryName && <h5 className="mb-3 mt-n4">More courses that teach you {subCategoryName} Skills</h5>}
```

Comment:
Added conditional to take into account if no subcategory name passed.

Reply:
[clarification] Why is it necessary to conditionally render based on truthy `subCategoryName`, as `subCategoryName` is a required prop?